BrainPortal/app/models/boutiques_portal_task.rb

```diff
@@ -591,6 +592,13 @@ def check_oneisrequired_group(group, descriptor = self.descriptor_for_after_form
     params_errors.add(group.name, " need at least one parameter set")
   end
 
+  def check_allornone_group(group, descriptor = self.descriptor_for_after_form)
+    members = group.members
+    are_set = members.select { |inputid| ! isInactive(descriptor.input_by_id inputid) }
```

Comment:
Personally I'd have written:

```ruby
num_unset = members.count { |inputid| isInactive(descriptor.input_by_id inputid) }
return if num_unset == 0 || num_unset == members.size # all, or none
```

That's because arrays have a .count() method that can take a true-false block

Reply:
ok, I thought of something like that, even though opted to keep checkers uniform

Reply:
done